packages/medusa/src/api/routes/admin/price-lists/index.ts

```diff
@@ -93,25 +93,143 @@ export const defaultAdminPriceListFields = [
 
 export const defaultAdminPriceListRelations = ["prices", "customer_groups"]
 
+/**
+ * @schema AdminPriceListRes
+ * type: object
+ * properties:
+ *   price_list:
+ *     $ref: "#/components/schemas/PriceList"
+ */
 export type AdminPriceListRes = {
   price_list: PriceList
 }
 
+/**
+ * @schema AdminPriceListDeleteBatchRes
+ * type: object
+ * properties:
+ *   ids:
+ *     type: array
+ *     items:
+ *       type: string
+ *       description: The IDs of the deleted Money Amounts (Prices).
+ *   object:
+ *     type: string
+ *     description: The type of the object that was deleted.
+ *     default: money-amount
+ *   deleted:
+ *     type: boolean
+ *     description: Whether or not the items were deleted.
+ *     default: true
+ */
 export type AdminPriceListDeleteBatchRes = {
   ids: string[]
   deleted: boolean
   object: string
```

Comment:
Not directly related to OAS and isn't necessarily something that should be looked into in this PR: seems like there is an inconsistency across different files in how this type of response is defined. In some places, the default value is set within the type definition (for example, `object: 'payment_collection'`) whereas in other places like here the type is generic. Maybe this is something to take into account if we want to automatically generate OAS comments in the future.

Reply:
Indeed. I believe that latest approach is to declare `object:` as a constant. The way to do it in OAS would be to declare `object` as the following:
```yaml
object:
  - type: string
  - description: The type of the object that was deleted.
  - enum: [money-amount]
  - default: money-amount
```